README.md

```diff
@@ -12,7 +12,7 @@ pipenv run pip install pip==18.0
 pipenv install
 pipenv shell
 ./manage.py migrate
-./manage.py loaddata sites cana
+./manage.py loaddata sites cana livy tolsoy
```

Comment:
```suggestion
./manage.py loaddata sites cana livy tolstoy
```
I may have typo'd that when I resolved the merge conflict

Reply:
no, it already there before :-)

Reply:
Good catch @jtauber. Fixed.